testdata/test_filter_by_comm.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+. $(dirname "$0")/common.sh
+set -ex
```

Comment:
Why not adding -o pipefail?

Reply:
I copied these scaffold codes from other steps. I'm not sure what the side effects of changing the set option would be.

Reply:
I think we can use this `set -euxo pipefail`. It should be no side effects .